.github/workflows/pax-cuda-121.yaml

```diff
@@ -80,6 +86,18 @@ jobs:
           | ROSETTA(pax) | ${{ needs.build-rosetta-pax.outputs.DOCKER_TAGS }} |
           EOF
 
+  publish:
+    if: github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && inputs.PUBLISH)
+    needs: [metadata, build-pax]
+    uses: ./.github/workflows/_publish_container.yaml
+    secrets: inherit
+    with:
+      SOURCE_IMAGE: ${{ needs.build-pax.outputs.DOCKER_TAGS }}
+      TARGET_IMAGE: upstream-pax
+      TARGET_TAGS: |
+        type=raw,value=latest,priority=1000
+        type=raw,value=nightly-${{ needs.metadata.outputs.BUILD_DATE }},priority=900
```

Comment:
This looks like it will overwrite the tag that's already there. Did we want to add another tag for `12.1`?

Reply:
Good catch :)